stackoverflow-dark.user.css

```diff
@@ -1424,7 +1424,7 @@ regexp("^https?:\/\/((?!area51)\w+\.)*?stackexchange\.com.*") {
   /* Assign */
   #answers .answer.accepted-answer {
     background: rgba(/*[[accepted-color-rgb]]*/, .5) !important;
-    border: 1px solid rgba(/*[[accepted-color-rgb]]*/, 1) !important;
+    border: 1px solid rgba(/*[[base-color-rgb]]*/, 1) !important;
```

Comment:
base color could be anything else, we dont want to separate the border color.

The opacity between bg and border should be enough to get a slightly brighter border.